Engines/Wine/Verbs/Tahoma/script.js

```diff
@@ -19,6 +19,10 @@ class Tahoma {
         const wizard = this.wine.wizard();
         const prefixDirectory = this.wine.prefixDirectory();
         const fontDirectory = this.wine.fontDirectory();
+        
+        if (!fileExists(fontDirectory)) {
```

Comment:
The method name sounds strange in this case. Should we add a dedicated method for folders? What do you think @plata @ImperatorS79?

Reply:
We can do that. Would basically be an alias then, right?

Reply:
Actually I'm not sure. If we decide to split the functionality into a `fileExists` and `directoryExists` method we could also test whether the given path is not a file/directory but instead a directory/file, depending on the regarded method.